modules/300-prometheus/hooks/detect_vpa_max.go

```diff
@@ -84,13 +84,26 @@ func calculateNodesCapacity(input *go_hook.HookInput) error {
 	totalPodsMemory := totalPodsCapacity * memoryPerPod
 	totalPodsCPU := totalPodsCapacity * milliCPUPerPod
 
-	// calculate prometheus usage
-	maxMem := resource.NewQuantity(totalPodsMemory/1, resource.BinarySI)
-	maxCPU := resource.NewMilliQuantity(totalPodsCPU/1, resource.DecimalSI)
+	// calculate prometheus usage by MustParse method
+	minMem := resource.MustParse("1000Mi")
```

Comment:
We don't want to calculate it the every iteration. Just put it to the `var` block as `resource.NewQuantity...`